core/examples/http_bench.js

```diff
@@ -16,43 +15,6 @@ const responseBuf = new Uint8Array(
 const promiseMap = new Map();
 let nextPromiseId = 1;
 
-const opRegistry = [];
-
-class Op {
-  constructor(name, handler) {
-    this.name = name;
-    this.handler = handler;
-    this.opId = 0;
-    opRegistry.push(this);
-  }
-
-  setOpId(opId) {
-    this.opId = opId;
-  }
-
-  /** Returns i32 number */
-  sendSync(arg, zeroCopy = null) {
-    const buf = send(0, this.opId, arg, zeroCopy);
```

Comment:
i think the idea behind this being a class method was the opId could be private (and perhaps not exposed)...

Reply:
Not really, not need to hide op id. Using class drastically lowers performance though.